prow/scripts/cluster-integration/helpers/cleanup-cluster.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
this script requires some env variables, please describe them at the beginning of the file. Add such comment also to other helpers you created

Reply:
Added comments on usage